openadapt/crud.py

```diff
@@ -109,28 +116,107 @@ def insert_recording(recording_data):
 
 
 def get_latest_recording():
-    return (
-        db
-        .query(Recording)
-        .order_by(sa.desc(Recording.timestamp))
-        .limit(1)
-        .first()
-    )
+    return db.query(Recording).order_by(sa.desc(Recording.timestamp)).limit(1).first()
+
+
+def get_recording_by_id(recording_id):
+    return db.query(Recording).filter_by(id=recording_id).first()
+
+
+def export_sql(recording_id):
+    """Export the recording data as SQL statements.
+
+    Args:
+        recording_id (int): The ID of the recording.
+
+    Returns:
+        str: The SQL statements to insert the recording into the output file.
+    """
+    engine = sa.create_engine(config.DB_URL)
+    Session = sessionmaker(bind=engine)
+    session = Session()
+
+    recording = get_recording_by_id(recording_id)
+
+    if recording:
+        sql = f"INSERT INTO recording VALUES ({recording.id}, {recording.timestamp}, {recording.monitor_width}, {recording.monitor_height}, {recording.double_click_interval_seconds}, {recording.double_click_distance_pixels}, '{recording.platform}', '{recording.task_description}')"
```

Comment:
Can you please format as follows:
```
        values = [
            recording.id,
            recording.timestamp,
            ...
            f"'{recording.task_description}'"
        ]
        values_str = ", ".join(values)
        sql = f"""
            INSERT INTO recording
           VALUES ({values_str})
        """
```
(Even better would be to load all the values dynamically from the table definition, but that may be overkill for now given the requirements around formatting different types)

Reply:
resolved in https://github.com/MLDSAI/OpenAdapt/pull/223/commits/9dc1850918e81648b9885dd8a1be836d5e8d95ef